ede-php-autoload/class-loader/psr0.el

```diff
@@ -125,17 +124,26 @@ PREFIX is the beginning of the type to complete."
                             namespace
                             "")))
       (setq suggestions (delete-dups
-                         (delete nil
-                                 (mapcar
-                                  #'(lambda (file-name)
-                                      (ede-php-autoload--create-psr-0-suggestions
-                                       file-name
-                                       prefix))
-                                  (ede-php-autoload--gather-relative-subfiles
-                                   directories
-                                   project-root
-                                   relative-path
-                                   (car (last split-prefix)))))))))
+                         (mapcar
+                          #'(lambda (file-name)
+                              (let ((suggestion (ede-php-autoload--create-psr-0-suggestions
+                                                 file-name
+                                                 prefix)))
+                                (ede-php-autoload--make-completion
+                                 suggestion
+                                 ;; A bit of a cop out, but right now it's the best option.
+                                 (if (string-match-p "\\.php$" file-name) :file :directory)
```

Comment:
Well, `file-name` is just the plain name, and `ede-php-autoload--gather-relative-subfiles` might have found the file in any number of directories.

Changed it to string-suffix-p, how do you like that?

(also, see follow up comment)